doorman/plugins/logs/base.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABCMeta, abstractmethod, abstractproperty
+
+from doorman.compat import with_metaclass
+
+
+class AbstractLogsPlugin(with_metaclass(ABCMeta)):
+    @abstractproperty
```

Comment:
Had this for some debugging, but not sure it's necessary - will revisit tomorrow.